enqueue/enqueue-bundle/0.9/config/packages/test/enqueue.yaml

```diff
@@ -0,0 +1,5 @@
+enqueue:
+    default:
+        transport: 'null:'
```

Comment:
`null:` or `null://`

Reply:
`null:` is a valid scheme.

Reply:
I know, but then we should use it here too:
https://github.com/symfony/recipes-contrib/pull/565/files#diff-d1033b7e1d7ea30fd6e9eae6a48948e8R9

I am against mixing it with or without `//`